_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.md

```diff
@@ -3,59 +3,61 @@ layout: post
+In our case, the termination points were specific branches in our repository. For example, all of the code prior to, and included in, a merge with the staging branch, was the responsibility of those focused on the application development work. Once the code was pushed to this branch, a deployment pipeline was responsible for deploying and hosting the application. This is noteworthy as the teams did not communicate through XaaS, per se, rather their work was coordinated through a termination point whereby the responsibilities were separated. This serves the same function as the XaaS in Figure 2. Namely, that a team's work can be provided to others through an automated service, whether that be XaaS, or through some agreed upon termination point. Enforcing this is critical, as without it, deployment on demand can never be realized. This behaviour and way of working is where standards are required. Standardizing automated communication (while not specifying specific tools) is crucial for scaling automation and achieving deployment on demand.
 
 For this project, a manager acted as the product owner for each of the teams, the Value Stream Aligned Team, the DTS Platform Team, and the second SE Platform Team. Please note this does not need to be the case. Actually, ideally it should not be the case. The product owner (the one responsible for prioritizing the work of the team) should be someone from business (albeit with technical knowledge or understanding). One model often referenced is the so-called, and poorly named, “Spotify model”, which is only used here as a demonstration of an alternate model whereby product owners and managers needn't be the same person. While this is generally the model that was used, it is abstracting away complexities that make it difficult for this model to work well in a government context. For example, the director at the top who is responsible for the management of the product is not the director for all of the teams in the box, though he is responsible for the platform, which makes reporting, hierarchies, and performance management a potential issue.
```

Comment:
In this paragraph we say that 'a manager' acted as the product owner but later say it would be better for the product owner to be someone from business.  But the first sentence isn't clear that the 'manager' wasn't from business.  Was it an IT manager?  I would find the rest of this paragraph more useful if it spoke to the reasons why having someone from business involved would be good, rather than the academics around other models.  If what we are saying here is that it would be better for the product manager to be someone from business but that this isn't possible in gc context, then I'm not sure this has value add rather than cognitive load....

Reply:
Not sure I want to get in to that here, as Im trying to focus on the model but I added a link to a document by 18F that speaks about the topic). 

Do you think that will suffice?